CHANGELOG.md

```diff
@@ -1,3 +1,119 @@
+## v0.7.0 - 2021-03-01
```

Comment:
Other than date change before merging this PR, LGTM

Reply:
Thanks, done.